nvflare/fuel/utils/config.py

```diff
@@ -118,8 +120,6 @@ def load_config(self, file_path: str) -> Config:
             return Config
         """
 
-        pass
-
```

Comment:
just curious, is having this one line `pass` clearer or not having it?
we have some places also got this pattern, we can update them as well.